src/Illuminate/Console/Concerns/ConfiguresPrompts.php

```diff
@@ -167,4 +168,48 @@ protected function restorePrompts()
     {
         Prompt::setOutput($this->output);
     }
+
+    /**
+     * Validate the given prompt.
+     *
+     * @param  mixed  $value
+     * @param  mixed  $rules
+     * @return ?string
+     */
+    protected function validatePrompt($value, $rules)
+    {
+        if (! $rules) {
+            return null;
+        }
+
+        $field = 'answer';
+
+        if (is_array($rules) && ! array_is_list($rules)) {
+            [$field, $rules] = [key($rules), current($rules)];
+        }
+
+        return Validator::make([$field => $value], [$field => $rules], $this->messages(), $this->attributes())
```

Comment:
```suggestion
        return $this->laravel['validator']->make([$field => $value], [$field => $rules], $this->messages(), $this->attributes())
```

Best to use available app instance instead of facade here.

Reply:
Great suggestion, thank you @crynobone 👍